include/Editor.h

```diff
@@ -57,6 +57,11 @@ class Editor : public QMainWindow
 	DropToolBar * addDropToolBar(QWidget * parent, Qt::ToolBarArea whereToAdd, QString const & windowTitle);
 
 	void closeEvent(QCloseEvent * event) override;
+	void keyPressEvent(QKeyEvent * ke) override;
```

Comment:
> For pointers or references, always use a single space between the type and '*' or '&', but no space between the '*' or '&' and the variable name

Source: [Qt Coding Style](https://wiki.qt.io/Qt_Coding_Style#Spaces_within_code) since it is not explicitly overidden in the LMMS coding conventions

```suggestion
	void keyPressEvent(QKeyEvent *ke) override;
```